src/interfaces/IStatefulERC712.sol

```diff
@@ -2,10 +2,8 @@
 
 pragma solidity 0.8.23;
 
-import { IERC712 } from "./IERC712.sol";
-
 /// @title Stateful Extension for EIP-712 typed structured data hashing and signing with nonces.
-interface IStatefulERC712 is IERC712 {
+interface IStatefulERC712 {
```

Comment:
If `IStatefulERC712` is a `Stateful Extension for EIP-712`, `IStatefulERC712` needs to extend `IERC712Domain`.

Reply:
Fixed in this commit: https://github.com/MZero-Labs/common/pull/7/commits/287b68343734e04236e9d631ee9a4d0e8db50a21